.sassdocrc

```diff
@@ -45,9 +45,7 @@ herman:
       url: https://oddbird.net/cascading-colors/
     - name: Herman
       url: https://oddbird.net/herman/
-  fontpath: src/fonts/
-  nunjucks:
-    templatepath: content/_includes/
+  customCSS: _built/css/styleguide.css
```

Comment:
This doesn't actually fix the problem, because `styleguide.scss` only really contains styles for CCS (why?). Changing this to `_built/css/screen.css` fixes it, but overrides way too many Herman styles.